packages/directory/addon/components/dir-asset-row-actions.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ * {{dir-asset-row-action
+ *   value=assetModel
```

Comment:
`row=rowObject`

Reply:
Custom cell components in ember light table require an attribute named `value` to pass arguments through

You define the column that will have the custom cell component and the table will pass the model through `value` to the cell component